src/cbottle/inference.py

```diff
@@ -1122,4 +1132,16 @@ def load(model: str, root="", **kwargs) -> CBottle3d:
             allow_second_order_derivatives=True,
             **kwargs,
         )
+
+    elif model == "cbottle-3d-moe-aimip":
+        checkpoints = "training-state-000512000.checkpoint,training-state-002048000.checkpoint,training-state-009856000.checkpoint".split(  # v1
+            # checkpoints = "training-state-000512000.checkpoint,training-state-002176000.checkpoint,training-state-009984000.checkpoint".split( # v2
+            # checkpoints = "training-state-000640000.checkpoint,training-state-002048000.checkpoint,training-state-010112000.checkpoint".split( # v3
+            # checkpoints = "training-state-000640000.checkpoint,training-state-002176000.checkpoint,training-state-009728000.checkpoint".split( # v4 and v5
```

Comment:
To align with the description we are using for AIMIP, let's use "p" (physics index) to denote the different chekcpoints. Also please add separate "models" for each ensemble member e.g. "cbottle-3d-moe-aimip-p1".